type/__libvirtd/manifest

```diff
@@ -0,0 +1,233 @@
+#!/bin/sh -e
+#
+# 2020-2023 Dennis Camera (dennis.camera at ssrq-sds-fds.ch)
+#
+# This file is part of skonfig-extra.
+#
+# skonfig-extra is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# skonfig-extra is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with skonfig-extra. If not, see <http://www.gnu.org/licenses/>.
+#
+
+version_ge() {
+	# usage: version_ge version_is min_version_expected
+	#
+	# compares two version numbers (up to three numeric components separated
+	# with dots)
+	# returns 0 if `version_is` >= `min_version_expected`, 1 otherwise.
+	#
+	# NOTES: if the lowest value (1st line) of the two version numbers is equal
+	#        to `min_version_expected`, `version_is` must be the same or larger.
+	#        explicit `-k n,n` is required for some sort(1)s.
+	printf '%s\n' "$1" "$2" \
+	| sort -t '.' -n -k 1,1 -k 2,2 -k 3,3 \
+	| head -1 \
+	| grep -qxF "$2"
+}
+
+os=$(cat "${__global:?}/explorer/os")
+os_version=$(cat "${__global:?}/explorer/os_version")
+mach_name=$(cat "${__global:?}/explorer/machine")
+init=$(cat "${__global:?}/explorer/init")
+
+: "${__type:?}"  # make shellcheck happy
+case ${os}
+in
+	(debian|devuan|ubuntu)
+		libvirtd_restart_cmd='/etc/init.d/libvirtd restart'
+
+		# NOTE: dnsmasq is required for virtual networks (as DHCP, DNS server)
+		__package dnsmasq
+
+		# NOTE: ebtables is required for some network filtering options
+		__package ebtables
+
+		# __update_alternatives iptables --path /usr/sbin/iptables-legacy
+
+		__package qemu-utils
+
+		# NOTE: We have to install netcat-openbsd because libvirt's developers
+		# are are complete idiots ignoring the fact that there are different
```

Comment:
let's not say bad things about people

Reply:
Whoops, looks like I didn't mean to publish this type when I wrote it 😄.